owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/OwnCloudClient.java

```diff
@@ -216,7 +213,7 @@ public RedirectionPath followRedirection(HttpBaseMethod method) throws Exception
                 redirectionsCount++;
 
             } else {
-                Log_OC.d(TAG + " #" + mInstanceNumber, "No location to redirect!");
+                Timber.d(" #" + mInstanceNumber + "No location to redirect!");
```

Comment:
This leading space makes no sense in my point of view

Reply:
but before `#` 
eg
` Timber.d("#" + mInstanceNumber + " No location to redirect!");`